src/main/views/accessibility-statement.njk

```diff
@@ -0,0 +1,86 @@
+{% extends "template.njk" %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+
+  {# Section 1 #}
```

Comment:
Could wrap sections in id'd divs to help section areas

Reply:
Good point, done :)